packages/portfolio-contract/src/portfolio.contract.ts

```diff
@@ -89,7 +96,8 @@ export const contract = async (
     {
       zoeTools,
       chainHubTools,
-      contract: privateArgs.contract,
+      contractAddresses,
+      axelarChainsMap,
     },
```

Comment:
These contexts are almost the same... I **suggest** we make it more clear how the're different:

```ts
const ctx1 = {
      zoeTools,
      chainHubTools,
      contractAddresses,
      axelarChainsMap,
    };
const { rebalance } = orchestrateAll(
    { rebalance: flows.rebalance },
    ctx1
  );
```